src/pages/Pool/CTACards.tsx

```diff
@@ -93,8 +94,9 @@ const ResponsiveColumn = styled(AutoColumn)`
 `
 
 export default function CTACards() {
-  const { chainId } = useWeb3React()
-  const { infoLink } = CHAIN_INFO[chainId ? chainId : SupportedChainId.MAINNET]
+  const { chainId, connector } = useWeb3React()
+  const { infoLink } = CHAIN_INFO[chainId && isChainAllowed(connector, chainId) ? chainId : SupportedChainId.MAINNET]
```

Comment:
To avoid this in the future, lets make a function called `getChainInfo` and then don't export `CHAIN_INFO`. this function should throw an error if the chainId isn't supported, which we'll be forced to catch.